.github/workflows/test-enos-scenario-ui.yml

```diff
@@ -100,15 +99,16 @@ jobs:
         if: contains(${{ github.event.repository.name }}, 'ent')
         run: echo "${{ secrets.VAULT_LICENSE }}" > ./enos/support/vault.hclic || true
       - name: Install Chrome Dependencies
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
         run: |
           sudo apt update
           sudo apt install -y libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev libasound2
       - name: Install Chrome
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
         uses: browser-actions/setup-chrome@v1
-      - run: |
-         chrome --version
+      - name: Chrome Version
```

Comment:
why did we need to separate this step ?

Reply:
I guess it's only useful to print the version of Chrome that is installed. Not sure that's logged anywhere else. Could be interesting to know that, if the UI tests fail for some weird reason, then you can test locally with that exact version of Chrome.